int32 goal_Id
int32 xmin
int32 xmax
int32 ymin
int32 ymax
int32 x
int32 y